verification/curator-service/api/src/model/user.ts

```diff
@@ -14,12 +14,17 @@ const userSchema = new mongoose.Schema({
         // supporting more identity platforms.
         required: 'User must be logged-in with Google',
     },
+    role: {
+        type: String,
+        // Note: not required as assuming no defined role === no permission
+    }
 });
 
 export type UserDocument = mongoose.Document & {
     googleID: string;
     name: string;
     email: string;
+    role: string;
```

Comment:
This should be an array of roles as one can have many roles.
We could use a hierarchy of roles and assume that admin implies curator implies reader but being explicit about each role is better imo.